modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -474,6 +474,9 @@
     /** */
     private boolean lateAffAssignment = DFLT_LATE_AFF_ASSIGNMENT;
 
+    /** */
+    private long clusterRegionId = Long.MIN_VALUE;
```

Comment:
I'm thinking it's more predictable behavior: all nodes with default configuration less or equal than nodes with region id.